jetty-start/src/main/java/org/eclipse/jetty/start/CommandLineBuilder.java

```diff
@@ -54,86 +55,200 @@ public static String findJavaBin()
     }
 
     /**
-     * Perform an optional quoting of the argument, being intelligent with spaces and quotes as needed. If a subString is set in quotes it won't the subString
-     * won't be escaped.
-     *
-     * @param arg the argument to quote
-     * @return the quoted and escaped argument
-     * @deprecated no replacement, quoting is done by {@link #toQuotedString()} now.
+     * @param arg string
+     * @return Quoted string
+     * @deprecated use {@link #shellQuoteIfNeeded(String)}
      */
     @Deprecated
     public static String quote(String arg)
     {
         return "'" + arg + "'";
```

Comment:
I know it's deprecated, but at least we can be consistent in Jetty 10.

```suggestion
        return shellQuoteIfNeeded(arg);
```

Reply:
I thought about that, but anybody that is calling this method is happy with the existing behavior. We should not change it.